spec/helpers/canonical_rails/tag_helper_spec.rb

```diff
@@ -13,10 +13,15 @@
 
   # Default behavior
   describe 'w/ default config' do
-    it 'should take the domain from the config' do
+    it 'should infer the domain name by looking at the request' do
       expect(helper.canonical_host).to eq 'www.alternative-domain.com'
     end
 
+    it 'should infer the domain name by looking at the request' do
```

Comment:
the label on this test is incorrect.  you're testing the port, not the domain name